apps/fluent-tester/src/TestComponents/CornerRadius/CornerRadiusTest.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { HOMEPAGE_CORNERRADIUS_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { Text } from '@fluentui/react-native';
+
+interface CornerRadiusTestComponentProps {
+  name: string;
+  cornerRadius: number;
+}
+
+const CornerRadiusTestComponent: React.FunctionComponent<CornerRadiusTestComponentProps> = (props: CornerRadiusTestComponentProps) => {
+  const outerBoxProps = React.useMemo(
+    () => ({
+      height: 60,
+      margin: 16,
+      backgroundColor: 'lightgrey',
```

Comment:
super nit: Maybe use a fluent color?

Reply:
Used colors from globalTokens.color.grey and also updated to swap colors in dark mode